pkg/clustermanager/kubetest2/gke.go

```diff
@@ -21,8 +21,8 @@ import (
 	"errors"
 	"fmt"
 	"log"
+	"os"
 	"os/exec"
```

Comment:
Since we are importing os couldn't we remove the `os/exec` import below and use `os.exec.*`?

Reply:
Not a blocker or anything just something to think about. It might be annoying to have `os.exec` everywhere.

Reply:
Golang does not support the `.*` syntax so we'll have to write them like this, see https://golangdocs.com/import-in-golang